state_transfer/endpoint/load_balancer/load_balancer.go

```diff
@@ -58,32 +59,42 @@ func (l *LoadBalancerEndpoint) Labels() map[string]string {
 	return l.labels
 }
 
-func (l *LoadBalancerEndpoint) Type() endpoint.EndpointType {
-	// endpoint type is not valid for this
-	return ""
+func (l *LoadBalancerEndpoint) IsEndpointHealthy(c client.Client) (bool, error) {
```

Comment:
Consider that this May stutter?

```go
 Endpoint := LoadBalancerEndpoint{}
 Endpoint.IsEndpointHealthy()
```

I think that IsHealthy may be acceptable?

Reply:
Yes, I was going to rename it, missed it, good catch!